packages/e2e-test-utils/README.md

```diff
@@ -364,6 +381,14 @@ _Returns_
 
 -   `Promise`: Promise resolving with current post content markup.
 
+### getEditedPageContent
```

Comment:
I think the name of this function is a bit ambiguous, it's not clear that it's the content of the site editor, a page can also be shown in the post editor. Maybe `getSiteEditorCurrentContent` or `getSiteEditorContent` or something like that?